sklearn/svm/_classes.py

```diff
@@ -1571,7 +1568,7 @@ def fit(self, X, y=None, sample_weight=None, **params):
         -----
         If X is not a C-ordered contiguous array it is copied.
         """
-        super().fit(X, np.ones(_num_samples(X)), sample_weight=sample_weight, **params)
+        super().fit(X, np.ones(_num_samples(X)), sample_weight=sample_weight)
```

Comment:
I think that we should have something like.

```suggestion
        if len(params) > 0:
            raise FutureWarning(
                "Passing additional keyword parameters has no effect and is "
                "deprecated in 1.0. An error will be raised from 1.2 and beyond."
                f"The ignored keyword parameter(s) are: {params.keys()}."
            )
        super().fit(X, np.ones(_num_samples(X)), sample_weight=sample_weight)
```

Reply:
We would need a test to check this behaviour.

Reply:
I have just added this changes. Let me know what you think.